web/src/engine/websites/ReadComicOnline.ts

```diff
@@ -31,7 +31,7 @@ const pageScript = `
                 tries++;
                 if (tries > 10) {
                     clearInterval(interval);
-                    reject(error);
+                    reject('Unable to get pictures after more than 10 tries !');
```

Comment:
Does this work?
I would expect to reject an `Error` type instead of a `string`, not sure how HaruNeko handles a thrown `string`.